### PR TITLE
Update rubygem-foreman_webhooks to 0.0.2

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* 2.4

### CODE REVIEW

**Comment by pcreech** [comment]
`Error: No Package found for tfm-npm(@theforeman/builder) < 5.0.0`

**Comment by pcreech** [comment]
[test rpm]

File: packages/plugins/rubygem-foreman_webhooks/rubygem-foreman_webhooks.spec [hunk 1]
```diff
@@ -33,8 +33,8 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 6.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 7.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.15.1
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 5.0.0
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_webhooks/rubygem-foreman_webhooks.spec:37]: this is not going to work, we have builder 8 now, can you please drop the upper bound, this seems to work fine for other plugins

Comment (lzap) [reply packages/plugins/rubygem-foreman_webhooks/rubygem-foreman_webhooks.spec]: Removed, thanks.

**Pull request merged**